docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1175,11 +1175,15 @@ You can also include a fragment with an `{#include}` section inside another temp
 
 <p>This document contains a detailed info about a user.</p>
 
-{#include item[item_aliases] aliases=user.aliases /} <1><2>
+{#include item$item_aliases aliases=user.aliases /} <1><2>
 ----
-<1> The `item[item_aliases]` parameter is translated as: _use the fragment `item_aliases` from the template `item`._
+<1> A template identifier that contains a dollar sign `$` denotes a fragment. The `item$item_aliases` value is translated as: _Use the fragment `item_aliases` from the template `item`._
 <2> The `aliases` parameter is used to pass the relevant data. We need to make sure that the data are set correctly. In this particular case the fragment will use the expression `user.aliases` as the value of `aliases` in the `{#for alias in aliases}` section.
 
+TIP: If you want to reference a fragment from the same template you can skip the part before `$`, i.e. something like `{#include $item_aliases /}`.
+
+NOTE: You can specify `{#include _ignoreFragments=true /}` in order to disable this feature, i.e. a dollar sign `$` in the template identifier does not result in a fragment lookup.
```

Comment:
```suggestion
NOTE: You can specify `{#include item$item_aliases _ignoreFragments=true /}` in order to disable this feature, i.e. a dollar sign `$` in the template identifier will not result in a fragment lookup.
```

Because otherwise it's confusing that we're not including any template :)